projects/storefrontlib/cms-structure/services/cms-features.service.ts

```diff
@@ -147,6 +180,25 @@ export class CmsFeaturesService {
     });
   }
 
+  protected registerOutletsForModule(moduleRef: NgModuleRef<any>) {
```

Comment:
This function is better run with `runModuleInitializersForModule` function in `LazyModulesService`. But, since all outlet codes are in `storefrontlib`, so I put it here.